### DISCUSSION TITLE
What are latest openAI models supported? can add gpt-4o in the list?

### DISCUSSION THREAD

**Discussion by sunyan-SG** [sinaptik-ai/pandas-ai#1261] [Q&A]
What are latest openAI models supported? can add gpt-4o in the list?  Thank you.

**Comment by gventuri** [comment]
Hi @sunyan-SG GPT4-o is already supported 😊

**Reply by sunyan-SG** [reply]
I just updated pandasAI and it can call gpt-4o now.  thank you very much.
